packages/l/libgnome-games-support/package.yml

```diff
@@ -1,8 +1,9 @@
 name       : libgnome-games-support
-version    : 1.8.2
-release    : 10
+version    : 2.0.0
```

Comment:
I look forward to this package splintering into a half dozen versions like libwebkit-gtk